api/src/test/java/org/pmiops/workbench/cohorts/CohortFactoryTest.java

```diff
@@ -0,0 +1,68 @@
+package org.pmiops.workbench.cohorts;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.pmiops.workbench.db.model.Cohort;
+import org.pmiops.workbench.db.model.CohortReview;
+import org.pmiops.workbench.db.model.User;
+
+import java.time.Clock;
+import java.util.Collections;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+
+public class CohortFactoryTest {
+
+    private CohortFactory cohortFactory;
+
+    @Before
+    public void setUp() {
+        cohortFactory = new CohortFactoryImpl(Clock.systemUTC());
+    }
+
+    @Test
+    public void createCohort() {
+        org.pmiops.workbench.model.Cohort apiCohort = new org.pmiops.workbench.model.Cohort();
+        apiCohort.setDescription("desc");
+        apiCohort.setName("name");
+        apiCohort.setType("type");
+        apiCohort.setCriteria("criteria");
+
+        User user = mock(User.class);
+
+        long workspaceId = 1l;
+
+        Cohort dbCohort = cohortFactory.createCohort(apiCohort, user, workspaceId);
+
+        assertThat(dbCohort.getDescription()).isEqualTo("desc");
```

Comment:
Nit: I might compare these against apiCohort rather than against the hardcoded string. Just to reduce a bit of brittleness.